Cargo.toml

```diff
@@ -16,5 +16,5 @@ winapi = { version =  "0.3.8", features = ["wincon", "winnt", "minwindef"] }
 crossterm_winapi = { version = "0.2.1" }
 
 [dependencies]
-crossterm_utils = { version = "0.3.1" }
+crossterm_utils = { version = "0.4.0" }
```

Comment:
I do see the reason behind this, but I prefer to increase version numbers when releasing. For example, when I increase all the versions with 1 I don't have to increase this one because it is already done. And that might create confusion.

Reply:
But we want/have to release `crossterm_utils` only, because we need them here. So, what do you suggest to make it working/passing?

Reply:
Maybe git dependency: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories. 

Although I am not sure if it chooses version over git or git over version.  If it would choose version over git, then this would work

Reply:
This wont add confusion? I mean, why we can't release & publish utils? It's still standalone crate.

Reply:
I am not sure if we talk about the same, are you talking about releasing utils now before merging this PR? So that it can pass travis?

Reply:
Not sure as well :) Yes that's what I mean.

Reply:
I'll do an experiment. I'm going to update my `-utils` PR (remove version bump, etc.), merge it to `master` and we'll see if it works with `git` & `version` deps.

Reply:
Seems to works locally, we'll see what it will do on Travis CI.

Reply:
Works. Here's the dependency line I used:

```toml
crossterm_utils = { git = "https://github.com/crossterm-rs/crossterm-utils.git", branch = "master", version = "0.3.1" }
```

Assuming you're okay with it as you proposed it, going to resolve this conversation and merge it.